frontend/src/components/SearchBar.jsx

```diff
@@ -13,10 +13,12 @@ import { ToastContainer, toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 
 // TODO: put in config file
-const serverURL = 'https://api-dot-nav-e-387904.uc.r.appspot.com/place';
+const serverURL = 'https://api-dot-nav-e-387904.uc.r.appspot.com/place?';
```

Comment:
Lol isn't there a better way to pass query params? This is pretty jank

Reply:
/place is my url endpoint: api.add_resource(Maps.Place, '/place')
the urlSearchParams object adds the /place$**placeID=asdf123** part. Most of the stack overflow posts just recommended using urlSearchParams for javascript.